modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/TrialFinishedNotes.kt

```diff
@@ -0,0 +1,116 @@
+package au.com.shiftyjelly.pocketcasts.profile
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.account.onboarding.upgrade.OnboardingUpgradeHelper.plusGradientBrush
```

Comment:
Can you move this brush somewhere else like `:modules:services:compose`? It's weird to have a profile feature use colors defined in onboarding. I recently added `PocketCastsColors` so it might fit there but I think you need to sync with `:main` first.

You could add similar extension to the `Brush` type.

https://github.com/Automattic/pocket-casts-android/blob/569c5d97ad5c26b237e74666a9220f634073b812/modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/PocketCastsColors.kt#L5-L21

Reply:
I liked that. I've moved this brush to this new class and I will make another PR to refactor the other classes that uses this brush